build.gradle

```diff
@@ -53,6 +118,7 @@ configure(subprojects.findAll { it.name.startsWith("maven-") }) {
     def mvnw = System.getProperty('os.name').toLowerCase(Locale.US).contains('windows') ?
             [ 'cmd', '/c', 'mvnw.cmd' ] : [ './mvnw' ]
 
+    mvnw += "-Dkotlin.version=$kotlin_version"
```

Comment:
Shouldn't we override all library versions as well? Or we only use ktor.version in maven examples?

Reply:
We use only kotlin and ktor dependencies in maven